tests/e2e/ctl_v3_elect_test.go

```diff
@@ -24,10 +24,6 @@ import (
 )
 
 func TestCtlV3Elect(t *testing.T) {
-	oldenv := os.Getenv("EXPECT_DEBUG")
```

Comment:
I assume these lines are removed because it is not much useful to print the extra debug lines?